### ISSUE TITLE
OMEMO indication

### ISSUE THREAD

**Issue by bugtrapper** [shtrom/Lurch4Adium#6]
With the build from June 7th, I don't get the (OMEMO) in the window title like my conversation partner does with Pidgin when we're in an OMEMO-encrypted conversation.
Is that possible with Adium at all?

**Comment by shtrom** [comment]
Yup, same thing here. Don't know if it's possible. Adium needs some wrapping around the libpurple plugins, and doesn't seem to be honouring the window title without more work.

Lurch does it in `lurch_topic_update_im` https://github.com/shtrom/libpurple-lurch/blob/macos-build/src/lurch.c#L2171, amongst others, called from `lurch_conv_created_cb`.

Ultimately, it uses `purple_conversation_set_title` https://github.com/shtrom/libpurple-lurch/blob/macos-build/src/lurch.c#L2184-L2186 . We'll need to work out how Adium handles this call, if at all.

**Comment by gkdr** [comment]
Well, this is very hacky anyway since Pidgin keeps changing the title every time a title gains or loses focus so I constantly have to check for a change event and change it back.
I basically only check if at least one session exists, maybe there is a more elegant way to do it in Adium?

**Comment by shtrom** [comment]
Yeah, I wonder whether we should just have the plugin write more info into the chat---say, when it starts, or once a day---in the same way as it flag unencrypted messages. This would be a change in Lurch, but it might work out.

**Comment by shtrom** [comment]
I think gkdr/lurch#129 should fix this issue. Need to test with that PR merged in a test branch.
